.github/workflows/sphinx_docx_to_gh_pages.yaml

```diff
@@ -26,6 +26,8 @@ jobs:
       - name: Running Sphinx to gh-pages Action
         uses: ns-rse/action-sphinx-docs-to-gh-pages@main
         with:
+          # When testing set this branch to your branch, when working switch to master. It WILL fail if not
+          # defined as it defaults to 'main'.
```

Comment:
This repo appears to have been started before GitHub switched the default `master` > `main`. It use `ns-rse/action-sphinx-docs-to-gh-pages@main` which I forked from the original ([action-sphinx-docs-to-gh-pages](https://github.com/uibcdf/action-sphinx-docs-to-gh-pages)) which itself defaults to `main`. I forked because it didn't use `sphinx-apidoc` to generate documentation from docstrings, but I got that working and have had the changes merged (see [here](https://github.com/uibcdf/action-sphinx-docs-to-gh-pages/pull/3) and [here](https://github.com/uibcdf/action-sphinx-docs-to-gh-pages/pull/4)).

Anyway, back on topic, I think defining `master` is fine and there will be fine times when people want to test the documentation builds on their branch but it shouldn't be blank/missing otherwise the build will fail as there is no `main` which is why I added the note.